capsules/extra/src/pressure.rs

```diff
@@ -0,0 +1,152 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright Tock Contributors 2023.
+
+//! Provides userspace with access to barometer sensors.
+//!
+//! Userspace Interface
+//! -------------------
+//!
+//! ### `subscribe` System Call
+//!
+//! The `subscribe` system call supports the single `subscribe_number` zero,
```

Comment:
This doesn't match the code

Reply:
Could you specify what specifically is not matching?

Reply:
I think this comment is right as-is, it matches: https://github.com/tock/tock/pull/3690/files#diff-4c9b3365129fd004838d073385ce262f5bac265f1829744c364bc9a07893d19fR127 I think.

Reply:
Ah I thought this was the command, not subscribe